classes/DB.php

```diff
@@ -1,13 +1,13 @@
 <?php
 /**
  * @file
- * Databse handling
+ * Database handling
  */
 
 class DB {
-  private static $qCounter = 0;
+  public static $qCounter = 0;
 
-  private static $history = array();
+  public static $history = array();
```

Comment:
Why change the access modifier here?
 
The commit message does not provide any context or reasoning. The `getHistory`method provides access.